### PR TITLE
added runtime input

### LINKED ISSUE #1247: Updating files to take input at runtime (#1160)
Updating these files :
Longest_Palindromic_Subsequence.cpp
Longest_Palindromic_Subsequence.java
Longest_Palindromic_Subsequence.py
Breadth_First_Search.cpp
Breadth_First_Search.java
Breadth_First_Search.py

### PR DESCRIPTION
**Pull request by prashant-mahanta**
Fixes #1247 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Modified below files for runtime input

#### Languages Used:
- C++
- Java
- Python

#### Files Added:
- Longest_Palindromic_Subsequence.cpp
- Longest_Palindromic_Subsequence.java
- Longest_Palindromic_Subsequence.py
- Breadth_First_Search.cpp
- Breadth_First_Search.java
- Breadth_First_Search.py

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java [hunk 1]
```diff
@@ -4,17 +4,20 @@
     of a given word which is a palindrome.
     A palindrome is a string that is same when read from any side.
 */
-public class Solution {
+import java.util.*;
+
+public class Longest_Palindromic_Subsequence {
```
[the remark this answers is not in the document]
Comment (prashant-mahanta) [reply Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java]: @SakshiShreya  Shouldn't be the file name and class name be same?

File: Breadth_First_Search/Breadth_First_Search.java [hunk 2]
```diff
@@ -60,25 +60,43 @@ void BFS(int s)
             }
         }
     }
+}
 
+public class Breadth_First_Search {
```
[the remark this answers is not in the document]
Comment (prashant-mahanta) [reply Breadth_First_Search/Breadth_First_Search.java]: @SakshiShreya Shouldn't be the file name and class name be same?

[the remark this answers is not in the document]
Comment (SakshiShreya) [reply Breadth_First_Search/Breadth_First_Search.java]: Yes, please change the file name also.

File: Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (SakshiShreya) [reply Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java]: Change the file name also.

File: Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.py [hunk 3]
```diff
@@ -52,22 +52,22 @@ def lps(st):
...
                     else:
-                        ans = lps_dp[i - 1][j]
+                        ans = lpsDp[i - 1][j]
                     temp.append(ans)
-        lps_dp.append(temp)
-    print("The longest palindromic subsequence is of length : ",lps_dp[length][length])
-    print("The longest palindromic subsequence is : ",print_lps(lps_dp, length, st, rev))
+        lpsDp.append(temp)
+    print("The longest palindromic subsequence is of length: ",lpsDp[length][length])
+    print("The longest palindromic subsequence is: ",printLps(lpsDp, length, st, rev))
```
Comment (SakshiShreya) [inline Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.py]: Add a space after `,`

File: Breadth_First_Search/Breadth_First_Search.cpp [hunk 4]
```diff
@@ -58,33 +58,48 @@ void Graph::bfs(int starting)
...
+4 0
+4 5
+5 6
+5 7
+6 2
+7 3
+
+Output:
 
 Breadth First Traversal is : 0 1 2 4 3 5 6 7
```
Comment (SakshiShreya) [inline Breadth_First_Search/Breadth_First_Search.cpp]: Remove the space before `:` in outputs also.

You can do a replace of all `<space>:` with `:` except in C++ files where you have used `::` for scope resolution

File: Breadth_First_Search/Breadth_First_Search.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (prashant-mahanta) [reply Breadth_First_Search/Breadth_First_Search.java]: Hey, I can't change the file name for this code as the BreadthFirstSearch.java already exists in this folder. I have added the runtime input and fixed other mistakes in the file.

**Comment by prashant-mahanta** [comment]
@SakshiShreya  I have changed as per requested except for one which I have mentioned earlier. Please review it.

**Pull request merged**